pygmt/_show_versions.py

```diff
@@ -109,22 +99,14 @@ def show_versions(file: TextIO | None = sys.stdout):
     It also warns users if the installed Ghostscript version has serious bugs or is
     incompatible with the installed GMT version.
     """
-
     sys_info = {
         "python": sys.version.replace("\n", " "),
         "executable": sys.executable,
         "machine": platform.platform(),
     }
-    dep_info = {
-        Requirement(v).name: _get_module_version(Requirement(v).name)
-        for v in importlib.metadata.requires("pygmt")  # type: ignore[union-attr]
-    }
-    dep_info.update(
-        {
-            "gdal": _get_module_version("osgeo.gdal"),
-            "ghostscript": _get_ghostscript_version(),
-        }
-    )
+    requirements = [Requirement(v).name for v in requires("pygmt")] + ["gdal"]  # type: ignore[union-attr]
```

Comment:
Previously, we have to use `_get_module_version("osgeo.gdal")`, now we can use `_get_module_version("gdal")`.

Reply:
Nice to see that `importlib.metadata.version("gdal")` works!